source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/row-actions.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+
+import { Dropdown, Icon } from 'semantic-ui-react';
+import { withTranslations, i18nProps } from '@lib/i18n';
+
+import { attributesFor } from '@data';
+import { ProjectAttributes } from '@data/models/project';
+
+import { withProjectOperations } from '@ui/routes/project/with-project-operations';
+
+export interface IProps {
+  project: JSONAPI<ProjectAttributes>;
+  toggleArchiveProject: (project: JSONAPI<ProjectAttributes>) => void;
```

Comment:
this method doesn't make a whole lot of semantic sense, imo.

`toggleArchiveProject`, but the entire project is being passed? is that necessary?

Also, I did a search for RowActions, and toggleArchiveProject isn't passed. :-\

Reply:
it's pass through the RowOperations HOC